cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/PartialRollingUpdateMockTest.java

```diff
@@ -84,37 +84,37 @@ public static void after() {
-                .withNamespace(NAMESPACE)
-                .build())
+                        .withNamespace(NAMESPACE)
+                        .build())
                 .withNewSpec()
-                    .withNewKafka()
-                        .withReplicas(5)
-                        .withListeners(new GenericKafkaListenerBuilder()
-                                    .withName("plain")
-                                    .withPort(9092)
-                                    .withType(KafkaListenerType.INTERNAL)
-                                    .withTls(false)
-                                    .build(),
-                                new GenericKafkaListenerBuilder()
-                                    .withName("tls")
-                                    .withPort(9093)
-                                    .withType(KafkaListenerType.INTERNAL)
-                                    .withTls(true)
-                                    .build())
-                        .withNewPersistentClaimStorage()
-                            .withSize("123")
-                            .withStorageClass("foo")
-                            .withDeleteClaim(true)
-                        .endPersistentClaimStorage()
-                    .endKafka()
-                    .withNewZookeeper()
-                        .withReplicas(3)
-                        .withNewPersistentClaimStorage()
-                            .withSize("123")
-                            .withStorageClass("foo")
-                            .withDeleteClaim(true)
-                        .endPersistentClaimStorage()
-                    .endZookeeper()
+                .withNewKafka()
+                .withReplicas(5)
+                .withListeners(new GenericKafkaListenerBuilder()
+                                .withName("plain")
+                                .withPort(9092)
+                                .withType(KafkaListenerType.INTERNAL)
+                                .withTls(false)
+                                .build(),
+                        new GenericKafkaListenerBuilder()
+                                .withName("tls")
+                                .withPort(9093)
+                                .withType(KafkaListenerType.INTERNAL)
+                                .withTls(true)
+                                .build())
+                .withNewPersistentClaimStorage()
+                .withSize("123")
+                .withStorageClass("foo")
+                .withDeleteClaim(true)
+                .endPersistentClaimStorage()
+                .endKafka()
+                .withNewZookeeper()
+                .withReplicas(3)
+                .withNewPersistentClaimStorage()
+                .withSize("123")
+                .withStorageClass("foo")
+                .withDeleteClaim(true)
+                .endPersistentClaimStorage()
+                .endZookeeper()
```

Comment:
The indentation is intentional to improve the readability. It should be reverted.

Reply:
Done